CITATION.cff

```diff
@@ -0,0 +1,166 @@
+
+repository-code: "https://github.com/bids-standard/stats-models.git"
+
+# identifiers:
+#   - description: This is the collection of archived snapshots of all releases of BIDSpm
+#     type: doi
+#     value: "10.5281/zenodo.3554331"
+
+contact:
+  - family-names: Markiewicz
+    given-names: Christopher J.
+    website: https://github.com/effigies
+    email: effigies@gmail.com
+  - family-names: de la Vega
+    given-names: Alejandro
+    website: https://github.com/adelavega
+
+authors:
+  # TEMPLATE
+  # - family-names: XXX
+  #   given-names: XXX
+  #   affiliation: XXX
+  #   website: XXX
+  #   email: XXX@XXX.XXX
+  #   orcid: "https://orcid.org/XXXX-XXXX-XXXX-XXXX"
+
+  #  Sorted alphabetically except for the 1rst and last authors
+
+  - family-names: Markiewicz
+    given-names: Christopher J.
+    website: https://github.com/effigies
+    email: markiewicz@stanford.edu
+    orcid: https://orcid.org/0000-0002-6533-164X
+    affiliation: Stanford University
+
+  - family-names: Bansal
+    given-names: Shashank
+    website: https://shashankbansal6.github.io/
+    email: shashankbansal56@gmail.com
+
+  - family-names: Blair
+    given-names: Ross W. 
+    website: https://github.com/rwblair
+
+  - given-names: Katherine
+    family-names: Bottenhorn
+
+  - family-names: Chen
+    given-names: Gang
+    website: https://github.com/afni-gangc
+    orcid: https://orcid.org/0000-0002-2960-089X
+    affiliation:
+      National Institutes of Health, Scientific and Statistical Computing Core,
+      NIMH, Bethesda, MD 20892, USA
+    email: gangchen@mail.nih.gov
+
+  - family-names: Dafflon
+    given-names: Jessica
+    website: https://jessicadafflon.github.io/
+
+  - family-names: Nielson
+    given-names: Dylan
+    affiliation:
+      Machine Learning Team at the National Institute of Mental Health
+    website: https://github.com/Shotgunosine
+
+  - family-names: Esteban
+    given-names: Oscar
+    website: http://www.axonlab.org
+    orcid: https://orcid.org/0000-0001-8435-6191
+    affiliation:
+      Department of Radiology, Lausanne University Hospital, Lausanne, CH-1011,
+      Switzerland
+    email: phd@oscaresteban.es
+
+  - family-names: Gau
+    given-names: Rémi
+    affiliation: Université catholique de Louvain
+    email: remi.gau@uclouvain.be
+    orcid: "https://orcid.org/0000-0002-1535-9767"
+
+  - family-names: Kent
+    given-names: James
+    affiliation:
+      Psychoinformatics Laboratory at the University of Texas at Austin
+    website: https://jdkent.github.io/
+    email: jamesdkent21@gmail.com
+
+  - given-names: Camille
+    family-names: Maumet
+    website: https://camillemaumet.com/
+    orcid: https://orcid.org/0000-0002-6290-553X
+    affiliation: Inria, Univ Rennes, CNRS, Inserm, Rennes, 35042, France
+
+  - family-names: Mumford
+    given-names: Jeanette
+    website: https://github.com/jmumford
+    email: jeanette.mumford@gmail.com
```

Comment:
@jmumford This will be used to credit you as a co-author of the BIDS Stats Models specification. Please confirm that you would like to be listed as an author and update the metadata as you see fit.

Reply:
Will Take the :+1: as an ok